ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -83,6 +83,10 @@
             // before initializing the form container load all the locale specific json resources
             // for runtime
             const formLanguage = formContainer.getLang();
+            if (formLanguage) {
```

Comment:
We would also need to do this on server, since this metadata helps search engines understand the language and text direction of the content. Can you open a ticket on backlog and assign it to @amit-mnnit ?